CONTRIBUTION.md

```
@@ -0,0 +1,429 @@
+
+   ```bash
+   cd streamlit_webrtc/frontend
+   pnpm dev
+   ```
+
+3. **Start Streamlit Application** (in a new terminal)
+   ```bash
+   uv run streamlit run home.py
+   ```
+
+The application will be available at http://localhost:8501
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)